ui/app/AppLayouts/stores/RootStore.qml

```diff
@@ -8,22 +8,198 @@ import StatusQ.Core.Theme 0.1
 import SortFilterProxyModel 0.2
 import AppLayouts.Profile.stores 1.0 as ProfileStores
 import AppLayouts.Wallet.stores 1.0 as WalletStore
+import AppLayouts.stores.Messaging 1.0 as MessagingStores
 
+// WIP: Previous reorganization step before refactoring `RootStore`
 QtObject {
     id: root
 
-    property var mainModuleInst: mainModule
-    property var walletSectionInst: walletSection
-    property var aboutModuleInst: aboutModule
-    property var communitiesModuleInst: communitiesModule
+    // Global properties that have to remain on `RootStore` (the module instances must be private properties and just used to initialize the
+    // rest and specific stores
+    readonly property bool isProduction: production
```

Comment:
I'm not sure this will work... it's an injected context property from NIM so it will likely not be available here